fluent/toInterfaceValue_test.go

```diff
@@ -0,0 +1,55 @@
+package fluent_test
+
+import (
+	"testing"
+
+	"github.com/ipld/go-ipld-prime/fluent"
+	"github.com/ipld/go-ipld-prime/node/basicnode"
+)
+
+func TestListValue(t *testing.T) {
```

Comment:
I'd encourage you to use table-driven tests, to avoid duplication. This way you can also have code coverage for all edge cases by just adding a few more entries to the table.

For instance: https://github.com/ipld/go-ipld-prime/blob/892d2f92ba8bdecef98d8863d310dbbd002b726c/node/bindnode/infer_test.go#L41